app/Http/Controllers/admin/ManageStudentController.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace App\Http\Controllers\admin;
+
+use App\Http\Requests\StoreStudent;
+use App\Http\Models\student;
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Http\Requests\LoginPost;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Pagination\LengthAwarePaginator;
+use Auth;
+class ManageStudentController extends Controller
+{
+    public function __construct()
+    {
+        return $this->middleware('guest');
+    }
+
+    protected function guard()
+    {
+        return Auth::guard('admin');
+    }
+    public function index(Request $request)
+    {
+        if($request->has('code_student')){
+            $CS = $request->input('code_student');
+            $students = student::where('code_student', 'like', "%{$CS}%")->paginate(5);
+            return view('admin.info_member', ['students' => $students]);
+        }
+        else{
+        $students = DB::table('student')->paginate(5);
+        return view('admin.info_member', ['students' => $students]);
+        }
+    }
+    public function sreach_student(request $request){
+
+    }
+/*
+    public function postLogin(LoginPost $request)
+    {
+
+        $email = $request->input('email');
+        $password = $request->input('password');
+        if (Auth::guard('admin')->attempt(['email' => $email, 'password' => $password])) {
+            $request->session()->put('user', 'admin');
+            return redirect()->intended('admin/home');
+        } else {
+            $errors = new MessageBag(['errorlogin' => 'Email hoặc mật khẩu không đúng']);
+            return redirect()->back()->withInput()->withErrors($errors);
+        }
+    }*/
+
+    /*public function logOut_Admin()
+    {
+        Auth::guard('admin')->logout();
+        return redirect('admin/login');
+    }*/
+    public function view()
+    {
+        return view('admin/create');
+    }
+
+    /**
+     * @param Request $request
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function store(StoreStudent $request)
+    {
+        $this->validate(request(), [
```

Comment:
@NguyenDucMinhBK  sao vẫn giữ lại $this->validate ?

Reply:
cháu xóa thiếu ạ.